src/views/LoginView.vue

```diff
@@ -1,40 +1,65 @@
-<template>
-  <v-app>
-    <v-main class="login-container">
-      <div>
-        <img src="@/assets/images/deltares_logo.png" alt="Deltares" contain style="max-width: 80%"/>
-      </div>
-      <h1 style="color: white;">Delft-FEWS Web Operator Client</h1>
-      <div class="login-providers">
-        <deltares-login name="Deltares"/>
-      </div>
-    </v-main>
-  </v-app>
-</template>
-
-<script lang="ts">
-import { Component, Vue } from 'vue-property-decorator'
-import DeltaresLogin from '@/components/DeltaresLogin.vue'
-
-@Component({
-  components: {
-    DeltaresLogin
-  }
-})
-export default class LoginView extends Vue {}
-</script>
-
-<style scoped>
-.login-container {
-  background-image: url("../assets/images/D111-103_getijdenland_small.jpg");
-  background-position-x: center;
-  background-size: cover;
-  text-align: center;
-  height: 100%;
-  overflow-y: auto;
-}
-
-.login-providers {
-  margin: 50px;
-}
-</style>
+<template>
+  <v-app>
+    <v-main class="login-container" id="web-oc-login-container-id">
+      <div>
+        <h1 style="color: white;">{{ title }}</h1>
+        <div class="login-providers">
+          <deltares-login loginButtonText="Sign in"/>
+        </div>
+      </div>
+    </v-main>
+  </v-app>
+</template>
+
+<script lang="ts">
+import { Component, Vue } from 'vue-property-decorator'
+import DeltaresLogin from '@/components/DeltaresLogin.vue'
+import {PiWebserviceProvider} from "@deltares/fews-pi-requests";
+
+@Component({
+  components: {
+    DeltaresLogin
+  }
+})
+export default class LoginView extends Vue {
+  webServiceUrl: string = ''
+  title: string = 'Delft-FEWS Web Operator Client'
+
+  async mounted(): Promise<void> {
+    const baseUrl = this.$config.get('VUE_APP_FEWS_WEBSERVICES_URL')
```

Comment:
this method is a bit long, it can be split in multiple methods, something like this

```
async initializeComponent(): Promise<void> {
  await this.setWebOcConfiguration()
  await this.setTitle()
  await this.setFavicon()
  await this.setBackgroundImage()
  document.title = this.title
```